roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -73,6 +73,7 @@ def run(self, tmp=None, task_vars=None):
             if r.get("failed", False):
                 result["failed"] = True
                 result["msg"] = "One or more checks failed"
+                result["changed"] = r.get("changed", False)
```

Comment:
Strange logic, why would we add the `"changed"` key only when a check failed?

We can *always* add `"changed"`, and compute it from the check results:

```python
result["changed"] = any(r.get("changed", False) for r in check_results.values())
```